### ISSUE TITLE
[Bug]: Small, low priority, warning text change - "Couldt" to "Could" in SnapshotManager.ts

### ISSUE THREAD

**Issue by NeilLamka** [homebridge-eufy-security/plugin#265]
### What happened?

Simple thing, in SnapshotManager.ts, line 326 you have a warning message that should be corrected.

The current message says 
 this.log.warn(this.device.getName(), 'Couldt not get cloud snapshot: ' + err);

and should read
 this.log.warn(this.device.getName(), 'Could not get cloud snapshot: ' + err);

by changing "Coudt" to "Could"

### Device Type

other (UnSupported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v12 (Supported)

### Operating System

Other (UnSupported)

### Relevant log output

```shell
[EufySecurity-2.1.4] WARN:  Deck Couldt not get cloud snapshot: Error: unable to verify the first certificate
[1/4/2023, 8:28:02 AM] [EufySecurity-2.1.4] WARN:  Deck snapshot handler is initialized without cloud snapshot. Maybe no snapshot will displayed the first times.
[1/4/2023, 8:28:02 AM] [EufySecurity-2.1.4] WARN:  Front Door Couldt not get cloud snapshot: Error: unable to verify the first certificate
[1/4/2023, 8:28:02 AM] [EufySecurity-2.1.4] WARN:  Front Door snapshot handler is initialized without cloud snapshot. Maybe no snapshot will displayed the first times.
[1/4/2023, 8:28:02 AM] [EufySecurity-2.1.4] WARN:  Driveway Couldt not get cloud snapshot: Error: unable to verify the first certificate
[1/4/2023, 8:28:02 AM] [EufySecurity-2.1.4] WARN:  Driveway snapshot handler is initialized without cloud snapshot. Maybe no snapshot will displayed the first times.
```

### config

```shell
{
            "platform": "EufySecurity",
            "username": "*************",
            "password": "************",
            "ignoreStations": [],
            "ignoreDevices": [],
            "country": "US",
            "cameras": [
                {
                    "serialNumber": "T8114P0220130947",
                    "enableCamera": true,
                    "enableButton": true,
                    "motionButton": true,
                    "rtsp": true,
                    "unbridge": true,
                    "forcerefreshsnap": false,
                    "useCachedLocalLivestream": false,
                    "videoConfig": {
                        "audio": true
                    }
                },
                {
                    "serialNumber": "T8114P0220130333",
                    "enableCamera": true,
                    "enableButton": true,
                    "motionButton": true,
                    "rtsp": false,
                    "unbridge": true,
                    "forcerefreshsnap": false,
                    "useCachedLocalLivestream": false,
                    "videoConfig": {
                        "audio": true
                    }
                },
                {
                    "serialNumber": "T8114P0220300F75",
                    "enableCamera": true,
                    "enableButton": true,
                    "motionButton": true,
                    "rtsp": true,
                    "unbridge": true,
                    "forcerefreshsnap": false,
                    "useCachedLocalLivestream": false,
                    "videoConfig": {
                        "audio": true
                    }
                }
            ],
            "cleanCache": true,
            "deviceName": "Beach Cameras"
        }
```

**Comment by NeilLamka** [comment]
I don’t see anything on the road map that seems to be related to my small defect. Is there one that I’ve missed?


Thanks Neil




Neil Lamka

Mobile:  206-931-0229<callto:206-931-0229>

***@***.******@***.***>

Sam Boni Clothing Company<http://www.sam-boni.com/>

________________________________

**Comment by NeilLamka** [comment]
Marking this as an enhancement vs as a defect  seems strange to me. That said, it does appear from the status and the issue being “stale”  the team is not interested in the details.

**Comment by lenoxys** [comment]
I am

**Comment by lenoxys** [comment]
in fact it was fixed in 2.2 release.
